### DISCUSSION TITLE
Use common text box for IP address , netmask , gateway so easier for copy paste

### DISCUSSION THREAD

**Discussion by hansko123** [snappysnappydog/Simple-IP-Config#123] [Ideas]
hi great job, I would like to suggest  something.

I want to be able to copy text containing IP address and just paste it to your Program. that way , it can be faster and less type error.

also have one click button for setting ip to DHCP.  

thanks

**Comment by KurtisLiggett** [comment]
Currently, if you click anywhere in the fields you can paste a complete IP in with right click or Ctrl+V.
You cannot copy an address though. In fact, if you try Ctrl+C it will clear all the fields! I should probably start with removing that shortcut. Then maybe add a button next to the IPs to copy.

**Comment by KurtisLiggett** [comment]
Copy and paste buttons have been added, available in the next release.

**Comment by KurtisLiggett** [comment]
Now available with release v2.9.7 - [Download Here](https://github.com/KurtisLiggett/Simple-IP-Config/releases/tag/2.9.7)
